File: src/examples/php/scanforms/create-scanform.php
<?php

require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';

use Dotenv\Dotenv;
use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\Scanform;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey($_ENV['EASYPOST_TEST_API_KEY']);

try {
    $shipments = ["shp_123..."];

    $scanform = ScanForm::create([
        "shipments" => $shipments,
    ]);

    echo $scanform;
} catch (Error $exception) {
    echo $exception;
}

File: src/examples/python/scanforms/create_scanform.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_TEST_API_KEY')

scan_form = easypost.ScanForm.create(shipments=['shp_123...'])

print(scan_form)
